README.md

```diff
@@ -65,6 +65,10 @@ If you are upgrading from 4.x.x to a current release, check out our [migration g
 If you use ProGuard or R8, you do not need to manually add any rules, as they are automatically embedded in the artifacts.
 Please let us know if you find any issues.
 
+## Development
+
+See [here](example-app/README.md) how to run the example app.
```

Comment:
Can we also explain a bit what this is? I think we shouldn't call it "example app" (despite the name 😅) to make sure it is seen as a testing app.